.gitignore

```diff
@@ -1,5 +1,7 @@
 .DS_Store
-.pyc
+*~
+*.svg
+*.pyc
```

Comment:
I can imagine making a case for adding `*.svg`, but I'm not sure what`*~` files are? If this isn't something many people will run into, adding a local exclusion to `.git/info/exclude` might be a better approach?

(BTW, the only reason I suggested reverting these was that in general, having separate PRs for independent changes seems to work well. But now that we've started, I'm happy to finalize this here.)